tools/statplot.py

```diff
@@ -236,34 +253,33 @@ def KeyPressed(self, widget, event):
     def PlotData(self, action, type, xscale, yscale):
         self.xData = self.values[..., self.xCombo.get_active()]
         self.yData = self.values[..., self.yCombo.get_active()]
-        if (numpy.unique(self.xData).size == 1
-                and numpy.unique(self.yData).size == 1):
-            type = 'marker'
         if action == 'create':
-            self.statplot, = self.ax.plot(self.xData, self.yData, linewidth=2,
-                                          color='xkcd:light purple')
+            self.statplot, = self.ax.plot(self.xData, self.yData)
```

Comment:
This `PlotData` method would be a little easier to read with a couple of blank lines (before the `if` lines and after the `elif` lines, for example)